languages/python/django-oso/tests/test_post_relationship.py

```diff
@@ -146,6 +146,9 @@ def test_authorize_scalar_attribute_condition(post_fixtures):
     """Scalar attribute condition checks."""
     Oso.load_str(
         """
+        banned_user(user: test_app2::User) if
```

Comment:
This should be split out into a separate test case.

Reply:
Why?

Reply:
The name/description of the test case is about scalar attributes, and this instead tests specializers on the sub field?

Would be good to have a separate specializer test, and should check that having multiple rules with different specializers works.

Reply:
Removed this from the test case in 182ac85

A test for this behavior exists as `test_partial_disjunctive_matches` in `django-oso/tests/test_django_oso.py`

Reply:
Tanks :)